packages/components/src/components/segmented-control/segmented-control-item/segmented-control-item-styles.ts

```diff
@@ -21,38 +24,36 @@ export const ICON_MARGIN = pxToRemWithUnit(4);
 export const getColors = (
   isDisabled: boolean,
   isSelected: boolean,
-  bgColor: SegmentedControlBackgroundColor,
   theme: Theme
-): { backgroundColor: string; buttonColor: string; labelColor: string } => {
+): {
+  buttonColor: string;
+  labelColor: string;
+  borderColor: string;
+  hoverBorderColor: string;
+} => {
   const themedColors = getThemedColors(theme);
-  const { primaryColor, contrastMediumColor } = isSelected ? getInvertedThemedColors(theme) : themedColors;
 
-  const backgroundColor =
-    themedColors[
-      isSelected ? 'contrastHighColor' : bgColor === 'background-surface' ? 'backgroundColor' : 'backgroundSurfaceColor'
-    ];
+  const { primaryColor, contrastMediumColor, disabledColor, contrastLowColor } = themedColors;
```

Comment:
Could be destructured directly now.

Reply:
@denyo I didn't get what you mean, could you give an example?

Reply:
@denyo I got it what you meant, changed it:

`const { primaryColor, contrastMediumColor, disabledColor, contrastLowColor } = getThemedColors(theme);`